src/drivers/intel/baytrail/ipc.c

```diff
@@ -125,39 +125,37 @@ static void ipc_platform_complete_cmd(void *data)
 	}
 }
 
-void ipc_platform_send_msg(struct ipc *ipc)
+void ipc_platform_send_msg(void)
```

Comment:
I'm not sure removing the ipc parameter and removing the `dsp_pending` flag are related changes that should remain in the same commit?